app/helpers/application_helper.rb

```diff
@@ -55,6 +55,10 @@ def enable_google_analytics(cookies)
       !cookies[CookiesController::NO_ANALYTICS_MODE]
   end
 
+  def using_non_primary_url?
+    ENV["PRIMARY_HOST"].blank? && ENV["PRIMARY_HOST"] != request.host
```

Comment:
This says "if PRIMARY_HOST is blank and is different to the host we're currently on, then we're using a non-primary URL." I don't think that's right.

Reply:
This checks if there is a primary host defined (returns out if not), and then, if we are not on it. (_I think 😬_). 

```
  def using_non_primary_url?
    return unless ENV["PRIMARY_HOST"].nil?

    ENV["PRIMARY_HOST"] != request.host
  end
```